hnn_core/gui/gui.py

```diff
@@ -671,25 +671,12 @@ def _get_connectivity_widgets(conn_data):
     for receptor_name in conn_data.keys():
         w_text_input = FloatText(value=conn_data[receptor_name]['weight'],
                                  disabled=False, continuous_update=False,
-                                 description="weight",
-                                 style=style)
-
-        w_slider = FloatLogSlider(value=conn_data[receptor_name]['weight'],
-                                  min=-5, max=1, step=0.2,
-                                  description=" ",
-                                  disabled=False,
-                                  continuous_update=False,
-                                  orientation='horizontal',
-                                  readout=False,
-                                  readout_format='.2e',
-                                  style=style)
-
-        link((w_slider, 'value'), (w_text_input, 'value'))
+                                 step=0.01, description="weight", style=style)
```

Comment:
shouldn't it be bounded to disable negative values?

Reply:
No, if we don't have a good upper bound for these variables. Because with `BoundedFloatText` we'll have a default `max` value to 100 even if we only specify the `min` to 0.

Reply:
just use a large value for `max` like 1e6 or something

Reply:
it's now updated. And I also updated some other `FloatText`s to `BoundedFloatText`s. Please help me to check if these changes are also valid or we need to move some of them back to `FloatText`.

Reply:
I reviewed the bounds you set in the `BoundedFloatText` objects, and everything looks good to me aside from my few comments below.

Reply:
For some step sizes, we don't have good estimates yet. So my strategy is to set the step size to 1 for things like `tsop` that involve ranges, and to 0.01 for other time-sensitive variables. This way, it will be easy to adjust them in the future if necessary. Let's stick with what we have now, as our main purpose in this PR is to change the input type.